test/Microsoft.AspNet.Mvc.IntegrationTests/Models/SoftwareViewModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.AspNet.Mvc.IntegrationTests
+{
+    [ModelMetadataType(typeof(Software))]
+    public class SoftwareViewModel
```

Comment:
Adding a lot of very similar model types with repeated use of the same attributes. Scale things back to highlight the important parts of the tests.